napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1319,6 +1320,255 @@ def process_mac_fields(vlan, mac, mac_type, interface):
 
         return mac_address_table
 
+    def _get_bgp_route_attr(self, destination, vrf, next_hop, ipv='v4'):
```

Comment:
Why is `ipv` a string and not a number?

Reply:
I noticed that the same approach is used in eos implementation. I decided to follow this way.  I agree that a number is probably better.

Thanks a lot for all your comments. I will update the code according to your recommendations.